src/utils/schemaEndpointResolver.js

```diff
@@ -29,12 +29,18 @@ function pathMatcher(routes, path) {
             return routeArr.every((seg, idx) => {
                 if (seg === pathArr[idx]) return true;
 
-                // if current path segment is param
-                if (seg.startsWith(':') && pathArr[idx]) return true;
+                if (!exactMatch) {
```

Comment:
wouldn't it be simpler to just do this iteration twice in this block, rather than introduce additional parameter and additional internal method?

Reply:
I think it's simpler to read the way I did it, but if you want me to refactor to do it here, just say the word :+1:

Reply:
nah, let's keep it as is